app/models/miq_worker/runner.rb

```diff
@@ -130,6 +133,11 @@ def start
     end
   end
 
+  def recover_from_provider_unreachable
+    sleep 2.minutes
```

Comment:
@blomquisg and I had discussed this and since there is 1 event catcher worker per provider we think it is OK to wait 2 minutes if the API to the providers seems to be failing.
@blomquisg Thoughts?